README.md

```diff
@@ -134,6 +134,26 @@ where
 
     Value associated with ANY key MAY be NULL.
 
+#### HW Capability Request
+Request:
```

Comment:
should follow the same structure and wording as "RC information".
Also add "HW Capability Request" to the supported "Mailbox requests"